src/axom/spin/MortonIndex.hpp

```diff
@@ -21,8 +21,7 @@
 #include "axom/core/Types.hpp"
 #include "axom/core/Macros.hpp"    // defines AXOM_STATIC_ASSERT
 
-#include "axom/primal/geometry/Point.hpp"
-#include "axom/primal/geometry/Vector.hpp"
+#include "axom/spin/Primitives.hpp"
 
 #ifdef AXOM_USE_CXX11
```

Comment:
I created #33 as a reminder to remove guards like ``#ifdef AXOM_USE_CXX11`` that deal with non-`C++11` compilers.

Reply:
Thanks.

I removed Primitives.hpp.